### PR TITLE
Use onSessionHandlers closes mediarain/generator-voxa-skill/#12

### PR DESCRIPTION
**Pull request by cvaca7**

### CODE REVIEW

File: generators/app/templates/skill/states.js [hunk 1]
```diff
@@ -6,6 +6,14 @@
 */
 
 exports.register = function register(skill) {
+  // This event is triggered after new session has started
+  skill.onSessionStarted((alexaEvent) => {});
+  // This event is triggered before the current session has ended
+  skill.onSessionEnded((alexaEvent) => {});
+  // This can be used to plug new information in the request
+  skill.onRequestStated((alexaEvent) => {});
```
Comment (rmberrios) [inline generators/app/templates/skill/states.js]: Misspell onRequestStated -> `onRequestStarted`

Comment (armonge) [inline generators/app/templates/skill/states.js]: there's a typo in here

**Review by rmberrios — CHANGES_REQUESTED** [review]

File: generators/app/templates/skill/states.js [hunk 2]
```diff
@@ -6,6 +6,14 @@
 */
 
 exports.register = function register(skill) {
+  // This event is triggered after new session has started
+  skill.onSessionStarted((alexaEvent) => {});
```
Comment (rmberrios) [inline generators/app/templates/skill/states.js]: Add reference to the each function http://voxa.readthedocs.io/en/latest/statemachine-skill.html

File: generators/app/templates/skill/states.js [hunk 3]
```diff
@@ -6,6 +6,14 @@
...
+  // This event is triggered before the current session has ended
+  skill.onSessionEnded((alexaEvent) => {});
```
Comment (rmberrios) [inline generators/app/templates/skill/states.js]: Please verify linting variable alexaEvent is not used

**Review by rmberrios — COMMENTED** [review]
Please use https://help.github.com/articles/closing-issues-via-commit-messages/

File: generators/app/templates/skill/states.js (hunk 3 above)
Comment (cvaca7) [reply generators/app/templates/skill/states.js]: yes, is on warning. I let alexaEvent on purpose for the user

**Review by rmberrios — APPROVED** [review]

**Pull request merged**